README.md

```diff
@@ -1,33 +1,58 @@
-# Project
-
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
-
-As the maintainer of this project, please make a few updates:
-
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
-
-## Contributing
-
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+# jfr-streaming 
+The jfr-streaming project provides a core library for configuring, starting, stopping, 
+and reading Java Flight Recording files from a JVM. The code does not depend on the `jdk.jfr`
+module and will compile and run against JDK 8 or higher. It uses a connection to an MBean
+server, which can be the platform MBean server, or a remote MBean server connected by
+means of JMX. 
+
+The goal of this project is a low-level library. Solving higher level problems, such
+as managing JFR across multiple JVMs, is not a goal of this project. 
+
+# Getting Started
+This example illustrates some of the API. 
+```java:    
+    public static void main(String[] args) {
+        MBeanServerConnection mBeanServer = ManagementFactory.getPlatformMBeanServer();
+        try {
+            FlightRecorderConnection flightRecorderConnection = FlightRecorderConnection.connect(mBeanServer);
+            RecordingOptions recordingOptions = new RecordingOptions.Builder().disk("true").build();
+            RecordingConfiguration recordingConfiguration = RecordingConfiguration.PROFILE_CONFIGURATION;
+            try (Recording recording = flightRecorderConnection.newRecording(recordingOptions, recordingConfiguration)) {
+                recording.start();
+                TimeUnit.SECONDS.sleep(10);
+                recording.stop();
+                recording.dump(Paths.get(System.getProperty("user.dir"), "recording.jfr").toString());
+            } catch (IOException ioe) {
+                ioe.printStackTrace();
+            } catch (InterruptedException ie) {
+
+            }
+        } catch (InstanceNotFoundException|IOException e) {
+            e.printStackTrace();
+        }
+    }
+```
+Note that, depending on the Java runtime, it may be necessary to unlock the Java Flight Recorder 
```

Comment:
That's Oracle JDK 8 only and we should explicitly mention that and also since 8u262 that JFR is now available for all OpenJDK distributions.